IdentityCore/src/webview/response/MSIDWebOAuth2AuthCodeResponse.m

```diff
@@ -55,33 +56,22 @@ - (instancetype)initWithURL:(NSURL *)url
         
         // populate auth code
         _authorizationCode = [NSString msidIsStringNilOrBlank:authCode] ? nil : authCode;
-        
-        // populate oauth error
-        _oauthError = oauthError;
     }
     
     return self;
 }
 
-+ (NSError *)oauthErrorFromParameters:(NSDictionary *)parameters
+- (MSIDAuthorizationCodeResult *)createAuthorizationCodeResult
 {
-    NSUUID *correlationId = [parameters objectForKey:MSID_OAUTH2_CORRELATION_ID_RESPONSE] ?
-    [[NSUUID alloc] initWithUUIDString:[parameters objectForKey:MSID_OAUTH2_CORRELATION_ID_RESPONSE]]:nil;
-    
-    NSString *serverOAuth2Error = [parameters objectForKey:MSID_OAUTH2_ERROR];
+    MSIDAuthorizationCodeResult *result = [MSIDAuthorizationCodeResult new];
+    result.authCode = self.authorizationCode;
+//    result.accountIdentifier = self.authCodeClientInfo.accountIdentifier;
+//    result.pkceVerifier = self.webViewConfiguration.pkce.codeVerifier;
```

Comment:
Shouldn't pkce be part of the base OAuth result?

Reply:
replied above, this commented code needs to be removed